doc/nrf/ug_thingy91.rst

```diff
@@ -170,9 +170,9 @@ Building and programming using |VSC|
 
 Complete the following steps after installing the |nRFVSC|:
 
-.. |sample_path_vsc| replace:: :file:`samples/nrf9160/cloud_client`
+.. |sample_path_vsc| replace:: :file:`samples/nrf9160/aws_iot`
```

Comment:
Why aws_iot for Visual Studio Code, and not Asset Tracker v2 that is used for command line?